packages/init/src/SliceMachineInitProcess.ts

```diff
@@ -920,7 +927,7 @@ ${chalk.cyan("?")} Your Prismic repository name`.replace("\n", ""),
 				title: `Creating new repository ${chalk.cyan(
 					this.context.repository.domain,
 				)} ...`,
-				task: async (_, task) => {
+				task: async (_, parentTask) => {
```

Comment:
💅 Could this be kept as `task` since there's no nested (parent) tasks here?

Reply:
Done 🙂